packages/react-front-kit/src/Assembly/DashboardCard/DashboardCard.stories.tsx

```diff
@@ -1044,3 +1050,54 @@ export const CardSimple: IStory = {
     </Card>
   ),
 };
+
+export const CardDataBadge: IStory = {
+  render: () => (
+    <Card radius={16}>
+      <Card.Section>
+        <CardHeader
+          leftSection={
+            <Note size={38} style={{ marginTop: '12px' }} weight="thin" />
+          }
+        >
+          <>
+            <h2 style={{ fontSize: '18px', margin: 0 }}>Element de text</h2>
+            <p style={{ margin: '0' }}>sous-text</p>
+          </>
+        </CardHeader>
+      </Card.Section>
+      <Card.Section m="32px 16px 16px">
+        <DataBadge
+          color="cyan"
+          number={48}
+          size="lg"
+          style={{ background: 'var(--mantine-primary-color-0)' }}
```

Comment:
La couleur de background doit aussi venir de la props `color` il faut que le composant soit le plus simple possible.

Reply:
Le background qui s'applique quand j'appli que le bon `cyan` au titre n'est pas le même `cyan` que  celui de la maquette c'est pour ça que je l'ai précisé ici.

Reply:
1. Ce qu'on dit dans les specs du ticket redmine prévaut, on n’intègre pas la maquette en pixel perfect.
2. Si tu penses que quelque chose qui es défini dans le ticket ne peut pas être appliqué ne prend pas de décision seul, remonte le point dans la discussion. Je préfère que le point soit clairement partagé et discuté plutôt que de découvrir que c'est pas fait comme c'était prévu lors de la revue de code.

Mais pour le coup ici on reste sur le point 1, tu utilises une shade de la couleur qui est passée en props qui se rapproche de la maquette et qui donne un contraste suffisant.

Reply:
C'est noté

Reply:
ça a un impact.
L'utilisation du composant n'est pas la même.
Elle est plus complexe dans ton cas.
Après ça ne change pas que si les couleurs que l'on défini de notre côté par défaut ne conviennent pas à l'utilisateur, il pourra toujours les surcharger en utilisant `style` par exemple.
Mais ça doit rester une utilisation spécifique et pas le cas normal.